class-network-mapping.php

```diff
@@ -73,7 +73,13 @@ public function is_active() {
 	 * @return stdClass|boolean {@see get_blog_details}
 	 */
 	public function get_network() {
-		return wp_get_network( $this->network );
+		global $wp_version;
+
+		if ( version_compare( $wp_version, '4.7', '<' ) ) {
```

Comment:
I think it's better to create separate function for wrapping this backward compatibility condition.